### DISCUSSION TITLE
Interest in developing the Design System!

### DISCUSSION THREAD

**Discussion by q1b** [Thinkmill/keystatic#154] [General]
Prior to this, myself Self-Taught Web Developer, Enthusiast and Keen interest in Web, Mostly as for a artist his painting are, for me coding is!

Skill-set that I gather around!

- Knowledge of Framework like, React, Svelte, Vue and Solidjs. among them, Have worked deeply with Solidjs and Vue
- CSS-Framework: Tailwindcss, since it's 2nd version, got introduced to it.
- Experience, Worked for a US-Based Company as Only Front-end Developer, to built a AI-Video Generator Platform, Pretty Advance Stuff here, is [link](https://www.bhuman.ai/)
- Knowledge of Using Github and Git Yes, need not to mention but Still.

Interested in contributing but, need some guide. little bit of help would bring a lot of clarity for me from, where should I start!

**Comment by JPrisk** [comment]
Hey @sacarvy,

If you would like to take Keystatic for a spin you can click one of the **Get Started** links from the [homepage](https://keystatic.thinkmill.com.au/). The blank canvas takes you through getting a basic Next.js + Keystatic site deployed to Vercel.

There is also a step by step instruction [here](https://thinkmill.notion.site/Automated-Setup-With-Template-76db9947899345ab8112deddf183d4aa). We are always keen to get feedback from the community, let us know how you get on.
